### PR TITLE
Feature/user removes ambassador/132100369

### PR DESCRIPTION
**Pull request by aviaweinstein**
Will need to test against the 'Feature/user-removes-ambassador/132100369' branches in the web app and store.

This branch fixes the style bugs at screen size 920px and it updates the ambassador nav dropdown manage link.

https://www.pivotaltracker.com/story/show/132100369

### CODE REVIEW

**Comment by aviaweinstein** [comment]
on hold so that we dont merge in before the store and web app prs are ready

File: app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns-ambassador.scss [hunk 1]
```diff
@@ -58,7 +58,9 @@ $your-ambassador-intro-text-width: 110px;
 }
 .dropdown-ambassador-edit_btn {
   display: block;
-  margin-top: -30px;
+  @media (min-width: $grid-float-breakpoint) {
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns-ambassador.scss]: prefer a return before a media query

File: app/assets/stylesheets/forever_style_guide/modules/_nav-offcanvas.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
 // https://github.com/iamphill/Bootstrap-Offcanvas
 $offcanvas-width:                 270px !default;
 $offcanvas-animation-time:        .15s !default;
-$offcanvas-media:                 $grid-float-breakpoint!default;
+$offcanvas-media-max:             $grid-float-breakpoint-max!default;
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-offcanvas.scss]: Do you think it's better to do away w/ this variable entirely?

**Comment by schweitzdesign** [comment]
Just a minor code style comment and a question about custom variables here, otherwise looks great.

**Comment by schweitzdesign** [comment]
Also an aside, in the future, I think it's preferable to have an explicit commit for the version bump.

File: app/assets/stylesheets/forever_style_guide/modules/_nav-offcanvas.scss (hunk 2 above)
Comment (aviaweinstein) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-offcanvas.scss]: yup, totally agree

**Comment by schweitzdesign** [comment]
Looks good to me!

**Pull request merged**